Documents/start_contribution.md

```diff
@@ -0,0 +1,128 @@
+# Contributing to JDI Light
+
+## 1: Setting Up the Repository for Contribution
+If you already have write access to the repository or do not need to create a personal fork for contribution, you can directly clone the repository to your local machine: 
+``` bash
+git clone https://github.com/jdi-testing/jdi-light.git
+```
+If you don't have write access, then first fork the repository to your own GitHub account by clicking the "Fork" button on the repository page. This creates a personal copy of the repository where you can make changes without affecting the original project. Next, clone your forked repository to your local machine using the command:
+
+``` bash
+git clone https://github.com/YOUR_USERNAME/jdi-light.git
+```
+
+Replace YOUR_USERNAME with your GitHub username. This step prepares your local development environment for making contributions.
+
+Insead cloning repo in a terminal you can do that by opening a project from IDE here an examples for different IDEs:
+
+- IntelliJ IDEA:
+    - If you see the Welcome screen, click on "Get from VCS".
+    - If you have already opened a project, select "File" > "New" > "Project from Version Control..." from the main menu.
+- Visual Studio Code:
+    - If you see the Welcome screen, click on "Clone Git repository".
+    - If you have already opened a project, press ctr+shift+G or select Source control in a left menu and push "Clone repository"
+
+
+## 2: Understand the project structure 
+
+Core methods and classes are resides in a jdi-light/jdi-light-core folder
+
+Project contains multiple folders like:
+- **.github/workflows** - in this folder you can find github actions description triggers on merging branches
+- Core methods and classes are resides in a **jdi-light/jdi-light-core** folder
+- **jdi-light-XXXX** - Contains the JDI Light specific module targets specific kind of testing or element base. You can see dependencies betveen modules in pom.xml for each module.
+- **jdi-light-XXXX-tests** - Holds tests for the abovementioned specific module. 
+- **jdi-light-XXXX-tests/src/test/resources** - holds variouce property files for example, the chrome.properties file is used to configure the Chrome driver, and the ff.properties file is used to configure the Firefox driver. 
+
```

Comment:
I think it is better to add a header that this is about browser and tests settings.
And also write that by default it must be headless to run in CI, locally it can be changed to see what happens

Reply:
added